Dronuts/client_src/src/components/employee/Menu.js

```diff
@@ -1,139 +1,74 @@
 import React, {Component} from 'react';
+import ItemAddForm from './ItemAddForm';
+import ItemEditForm from './ItemEditForm';
+import axios from 'axios';
 
-var menu = [
-  {
-    "name" : "Donuts",
-    "items" : [
-      {
-        "name" : "Plain Donut",
-        "description" : "Good old plain donut, probably the healthiest.",
-        "price" : 0.80
-      },
-      {
-        "name" : "Glazed Donut",
-        "description" : "Because plain donuts are boring, and diabetes is not a problem.",
-        "price" : 1.00
-      }
-    ]
-  },
-  {
-    "name" : "Beverages",
-    "items" : [
-      {
-        "name" : "Coffee",
-        "description" : "Can't survive CMU without this.",
-        "price" : 1.00
-      }
-    ]
+class Menu extends Component {
+  constructor(){
+    super();
+    this.state = {
+      menu: []
+    };
+  }
+
+  componentDidMount(){
+    axios.get('/api/Items')
+      .then(response => {
+        this.setState({menu: response.data});
+      });
   }
-];
 
-function renderItems(category){
-  return (
-    category.items.map((item) => (
+  deleteItem(name, id){
+    if (window.confirm('Are you sure you want to delete ' + name + '?')) {
+      axios.delete('/api/Items/' + id)
+        .then(response => {
+          window.location.reload();
+        });
+    }
+  }
+
+  render(){
+    const menuRows = this.state.menu.map((item) => (
       <div className="row">
-        <div className="col-4 border">
+        <div className="col-3 border">
           {item.name}
+          <ItemEditForm id={item.id} attribute="name" />
         </div>
-        <div className="col-6 border">
+        <div className="col-5 border">
           {item.description}
+          <ItemEditForm id={item.id} attribute="description" />
         </div>
         <div className="col-2 border">
           ${item.price}
+          <ItemEditForm id={item.id} attribute="price" />
         </div>
-      </div>
-    ))
-  )
-}
-
-function renderCategories(menu){
```

Comment:
Why are we removing categories? Are we no longer offering drinks as well?

Reply:
Because there isn't a notion of categories in our data models, so we can't separate items into categories.